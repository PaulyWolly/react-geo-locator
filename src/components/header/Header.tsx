import React, { FC, ReactElement } from 'react';
import { CompassTwoTone } from '@ant-design/icons'

import './header.styles.scss';
const Header: FC = (): ReactElement => {
  return (
    <div className="header">
      <h3>
        Ge<CompassTwoTone twoToneColor="#eb2f96" />Hospital
      </h3>
      <h4>Find nearest hospitals around your location</h4>
    </div>
  )
}

export default Header;