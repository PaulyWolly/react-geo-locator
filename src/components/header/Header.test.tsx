import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Header from './Header';

afterEach(cleanup);

test('Renders <Header /> component', () => {
  const { asFragment, getByText } = render(<Header />);
  const linkElement = getByText(/Find nearest hospitals around your location/i);
  expect(asFragment()).toMatchSnapshot();
  expect(linkElement).toBeInTheDocument();
});
