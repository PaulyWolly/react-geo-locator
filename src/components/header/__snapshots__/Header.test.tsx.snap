// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <Header /> component 1`] = `
<DocumentFragment>
  <div
    class="header"
  >
    <h3>
      Ge
      <span
        aria-label="compass"
        class="anticon anticon-compass"
        role="img"
      >
        <svg
          aria-hidden="true"
          class=""
          data-icon="compass"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M512 140c-205.4 0-372 166.6-372 372s166.6 372 372 372 372-166.6 372-372-166.6-372-372-372zM327.6 701.7c-2 .9-4.4 0-5.3-2.1-.4-1-.4-2.2 0-3.2L421 470.9 553.1 603l-225.5 98.7zm375.1-375.1L604 552.1 471.9 420l225.5-98.7c2-.9 4.4 0 5.3 2.1.4 1 .4 2.1 0 3.2z"
            fill="#fff0f6"
          />
          <path
            d="M322.3 696.4c-.4 1-.4 2.2 0 3.2.9 2.1 3.3 3 5.3 2.1L553.1 603 421 470.9l-98.7 225.5zm375.1-375.1L471.9 420 604 552.1l98.7-225.5c.4-1.1.4-2.2 0-3.2-.9-2.1-3.3-3-5.3-2.1z"
            fill="#eb2f96"
          />
          <path
            d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
            fill="#eb2f96"
          />
        </svg>
      </span>
      Hospital
    </h3>
    <h4>
      Find nearest hospitals around your location
    </h4>
  </div>
</DocumentFragment>
`;
